app/js/lib/foxbox/settings.js

```diff
@@ -89,8 +87,10 @@ export default class Settings extends Model {
 
   set localOrigin(origin) {
     this._localOrigin = origin ?
-        (new URL(origin)).origin : DEFAULT_LOCAL_ORIGIN;
-    storage.setItem(`${PREFIX}localOrigin`, this._localOrigin);
+        (new URL(origin)).origin : undefined;
+    if (this._localOrigin) {
+      storage.setItem(`${PREFIX}localOrigin`, this._localOrigin);
+    }
```

Comment:
Yes sorry, my wording was really awkward. @azasypkin got it right, do we need to call `removeItem` if the value is falsey?